<ui version="4.0" >
 <class>ServicesPage</class>
 <widget class="QWidget" name="ServicesPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Define Services</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QRadioButton" name="m_addWebServicesButton" >
     <property name="whatsThis" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If checked the next page ask you the list of WSDL used to known params of the WebServices.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="text" >
      <string>I want add some WebServices</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QRadioButton" name="m_noWebServicesButton" >
     <property name="whatsThis" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Don"t manage web services in the project.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="text" >
      <string>I don't want WebServices</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
